web/views.py

```diff
@@ -5,6 +5,7 @@
 from django.contrib.auth import logout as auth_logout
 from api.models import Project
 from api.models import UserProfile
+from api.models import Task
```

Comment:
I know we haven't talked about this one before and it's not really part of your PR, but the coding-style was updated, so I'd be grateful if you could join the the three imports from api.models into one line... (of course, try it out first :angel: )
so instead of
```
from api.models import Project
from api.models import UserProfile
from api.models import Task
```
this
`from api.models import Project, UserProfile, Task`

this should (only) be done when the imports are from the same module